scripts/clatd@.systemd

```diff
@@ -0,0 +1,22 @@
+#
+# clatd service file for systemd
+#
+# Install it to:  /etc/systemd/system/clatd.service
+# Enable it with: systemctl enable clatd.service
+# Start it with:  systemctl start clatd.service
```

Comment:
Are you certain that this is correct? Also, shouldn't `Makefile` install it to where it needs to go?